third_party/terraform/utils/config.go.erb

```diff
@@ -60,6 +60,10 @@ import (
 	"google.golang.org/api/storagetransfer/v1"
 )
 
+type providerMeta struct {
+	ModuleKey string `cty:"module_key"`
```

Comment:
I'm curious where this name came from. Why module_key and not just module, or module_name? @megan07

Reply:
@danawillow - i can make it whatever, when paddy and i talked about it he just threw out a name I could start with and get feedback on.  i'm happy to change it to module_name.